benchmark/src/androidTest/kotlin/org/dif/DIDCommBenchAndroidNaive.kt

```diff
@@ -0,0 +1,131 @@
+package org.dif.benchmark
+
+import android.util.Log
+import org.junit.jupiter.api.Disabled;
+import kotlin.test.Test
+
+import org.dif.fixtures.BenchCommon
+import org.dif.fixtures.BenchRes
+
+
+// TODO looks like that doesn't work in android
+@Disabled("Benchmark disabled for CI")
+class DIDCommBenchmarkAndroidNaive {
+
+    companion object {
+       private const val log_tag = "BenchLog"
+    }
+
+    //@get:Rule
+    //val benchmarkRule = BenchmarkRule()
+
+    fun measure(aFun: () -> Unit, N : Int = 1000, aFunName : String = "noname") {
+        val benchRes : BenchRes = BenchCommon.measure_naive(aFun, N, aFunName)
+        Log.i(log_tag, benchRes.toStr())
+    }
+
+    @Test
+    fun bench_pack_signed() {
+        // TODO resolve fun name dynamically
+        measure(
+            BenchCommon::pack_signed,
+            aFunName = "pack_signed"
+        )
+        /*
```

Comment:
Can we remove that commented lines?

Reply:
the intention was to keep some traces of [Android benchmark native framework](https://developer.android.com/jetpack/androidx/releases/benchmark) usage for the future